src: Rename jobs directories to match job names

alexjh opened the pull request:
Commit taken from uaa-fissile-release:994e67413ab26a072dd1d20d226cb875da13a713

New BOSH appears to validate that the job names match the directory they
reside in. Seems like a reasonable check.

Commit:
src: Rename jobs directories to match job names

Commit taken from uaa-fissile-release:994e67413ab26a072dd1d20d226cb875da13a713

New BOSH appears to validate that the job names match the directory they
reside in. Seems like a reasonable check.

mook-as reviewed (changes requested):
`cf-acceptance-tests` needs a bump; please check if I've missed other submodules? I don't _think_ I have, but I'm a little lost with all the submodules.

Commit:
CATS: Pull in changes to unbreak CATS

Originally from scf:36f704f2b648f8cfce6cd8c4808c188ce9c9e445

We don't have container networking yet

alexjh commented:
No, that looks like the only one that changed under `src/scf-helper-release` in https://github.com/SUSE/scf.

alexjh commented:
In hindsight, what I should have done was quickly split off the submodule with no changes, made the PR into SCF, then made a new PR for UAA and its changes. :confused:
